src/help.erl

```
@@ -0,0 +1,206 @@
+%% -------------------------------------------------------------------
+%%
+%% The main shell runner file for riak_shell
+%%
+%% Copyright (c) 2007-2016 Basho Technologies, Inc.  All Rights Reserved.
+%%
+%% This file is provided to you under the Apache License,
+%% Version 2.0 (the "License"); you may not use this file
+%% except in compliance with the License.  You may obtain
+%% a copy of the License at
+%%
+%%   http://www.apache.org/licenses/LICENSE-2.0
+%%
+%% Unless required by applicable law or agreed to in writing,
+%% software distributed under the License is distributed on an
+%% "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+%% KIND, either express or implied.  See the License for the
+%% specific language governing permissions and limitations
+%% under the License.
+%%
+%% -------------------------------------------------------------------
+-module(help).
+
+-export([
+         help/3,
+         help/1
+        ]).
+
+help(Mod, Example, List) ->
+    Msg1 = io_lib:format("The following functions are available~n", []),
+    Msg2 = print_exts(List),
+    Msg3 = io_lib:format("~nYou can get more help by calling help with the~n" ++
+                  "extension name and function name like 'help ~s ~s;'~n",
+              [Mod, Example]),
+    Msg4 = io_lib:format("~nFor SQL help type 'help SQL'", []),
+    Msg1 ++ Msg2 ++ Msg3 ++ Msg4.
+
+help('SQL') ->
```

Any chance we could make this case-insensitive? Or add a "not found" message when the match failed?  Now it just silently fails.

---

Made all commands case insensitive